script/plugin_webpack_directories.rb

```diff
@@ -26,14 +27,18 @@
   end
 
   path = "#{dep.to_spec.full_gem_path}/webpack"
-  entry = "#{path}/index.js"
-  # some plugins share the same base directory (tasks-core and tasks, REX etc)
-  # skip the plugin if its path is already included
-  next if config[:paths].include?(path)
-  if File.exist?(entry)
-    config[:entries][bundle_name dep.name] = entry
-    config[:paths] << path
+
+  Dir["#{path}/*index.js"].each do |entry|
+    # some plugins share the same base directory (tasks-core and tasks, REX etc)
+    # skip the plugin if its path is already included
+    next if entry_paths.include?(entry)
```

Comment:
I can't because `config[:paths]` aggregate paths to directories that contain entries while `entry_paths` contain paths to the actual entries.

e.g.
```
config[:paths]
 => [
  './katello/webpack/'
]
```
vs.
```
entry_paths
 => [
  './katello/webpack/index.js',
  './katello/webpack/common_index.js'
]
```